flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this remain very disappointing. why not building from source?

Reply:
Right now aarch64 is not a priority for me. I also don't have the hardware to test on aarch64 linux.